### ISSUE TITLE
Pretty-print "Notes" JSON

### ISSUE THREAD

**Issue by danvk** [hammerlab/cycledash#390]
It looks like the "Notes" field is mostly being used to hold JSON metadata about runs:

![screen shot 2014-12-15 at 12 21 48 pm](https://cloud.githubusercontent.com/assets/98301/5440264/00dfbc6c-8455-11e4-957c-0e543a268097.png)

If the "Notes" field parses as JSON, we should pretty-print it with syntax highlighting. This can be easily done using `JSON.stringify(obj, null, '   ')` and [highlight.js](https://highlightjs.org/).

**Comment by smondet** [comment]
JSon is fine for now but we could even define some conventions on the contents of this field?

Maybe with another field `params_format`?

For example (with ML notations):

```ocaml
type notes_format =
| Raw (* we don't know anything, we just display in a `<pre><core>` *)
| Raw_json (* we know it's Json *)
| Html  (* inline XSS vuln :) *)
| Biokepi_result (* what hammerlab/biokepi posts to cycledash *)
| (* etc. *)
```

**Comment by hammer** [comment]
Yeah I think some of these fields should be parsed and added as their own columns if they're going to be set consistently.

**Comment by ihodes** [comment]
@smondet Are there a few fields you'd like to have us move out, or just `params_format`?

Some sort of structure would definitely be useful here, both for display & for querying.

**Comment by smondet** [comment]
@ihodes the `params_format` with complex enough formats is maybe more future-proof than adding new fields (?)
